brianlong/www.validators.app
# 186674570 - add skipping processing of gifs

#### What's this PR do?
- Adds logic when processing animated gifs for avatars

#### How should this be manually tested?
- `rails test`
- 1) Create a validator with avatar_url included. Use some **GIF** sample - eg. "https://upload.wikimedia.org/wikipedia/commons/c/c0/An_example_animation_made_with_Pivot.gif"
- - in rails console utilize the validator and run `UpdateAvatarFileService.new(validator).call`. Check if the avatar has been attached (`validator.avatar.attached?`), it shouldn't be
- 2) Create a validator with avatar_url included. Use some **JPG** sample - eg. "https://s3.amazonaws.com/keybase_processed_uploads/3af995d21a8fe4cec4d6e83104f87205_360_360.jpg"
- - in rails console utilize the validator and run `UpdateAvatarFileService.new(validator).call`. Check if the avatar has been attached (`validator.avatar.attached?`). In this scenario the avatar should be successfully attached.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186674570)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
